### PR TITLE
Price feeds operational update

### PR DESCRIPTION
**Pull request by rustboyar**

### CODE REVIEW

**Review by sakulstra — APPROVED** [review]

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 1]
```diff
@@ -11,19 +11,24 @@ created: 2023-05-04
...
 
 ## Motivation
 
 The current price feeds for the LSTs can result in artificial volatility due to de-synchronization between different asset/PEG feeds. The same issue applies to the `WBTC` on Aave v2.
 To address this problem and enhance the stability of the Aave platform, we propose to change the price feeds for wstETH to a [custom price adapter](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CLSynchronicityPriceAdapterPegToBase.sol) that calculates the `wstETH / ETH / USD` price and is based on Chainlink's feeds under the hood.
 
-This approach is already used for the wstETH on Polygon and WBTC on Aave V3.
+WBTC price feed will use the same [adapter implementatin](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CLSynchronicityPriceAdapterPegToBase.sol), but with the `WBTC / BTC / ETH` inside.
+
+cbETH price feed will be swapped for an [special](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CbEthSynchronicityPriceAdapter.sol), which uses on-chain `cbETH / ETH` rate.
```
Comment (sakulstra) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:26]:
```suggestion
cbETH price feed will be swapped for an [special adapter](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CbEthSynchronicityPriceAdapter.sol), which uses on-chain `cbETH / ETH` rate.
```

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 2]
```diff
@@ -1,20 +1,30 @@
 ---
-title: Update price feeds for WBTC on Aave V2 and wstETH on Aave V3 Optimism and Arbitrum
+title: Price feeds operational update
 author: BGD Labs (@bgdlabs)
-shortDescription: Swap WBTC price feed on Aave V2 and wstETH feed on Aave V3 Optimism and Arbitrum
-discussions: https://governance.aave.com/t/bgd-generalised-price-sync-adapters/11416
+shortDescription: Swap price adapters for wbtc and LSTs
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:4]:
```suggestion
shortDescription: Swap price adapters for wbtc and LSTs, and activation of optimism price oracle sentinel
```

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 3]
```diff
@@ -1,20 +1,30 @@
...
+discussions: https://governance.aave.com/t/bgd-operational-oracles-update/13213
 created: 2023-05-04
 ---
 
 ## Simple Summary
 
-This proposal changes the current price feed for WBTC to the custom WBTC / BTC / ETH price adapter on Aave V2 Ethereum. wstETH price feeds are swapped for the custom wstETH / ETH / USD adapters on Aave V3 Optimism and Arbitrum. It also sets the price oracle sentinel on Aave V3 Optimism.
+This proposal changes the following price adapters:
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:11]: Add also price oracle sentinel

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 4]
```diff
@@ -1,20 +1,30 @@
...
+
+- WBTC on Ethereum v2 to the custom `WBTC / BTC / ETH`
+- wstETH on Optimism and Arbitrum v3 to `wstETH / ETH / USD`
+- MATICX on Polygon v3 to the custom adapter, which uses `MATIC / USD` CL feed and `MATICX / MATIC` on-chain rate
+- stMATIC on Polygon v3 to the custom adapter, which utilizes `MATIC / USD` CL feed and `stMATIC / MATIC` on-chain rate
 
 ## Motivation
 
-The current price feeds for wstETH on Aave V3 Optimism and Arbitrum (`wstETH/USD`) can result in artificial volatility due to de-synchronization between different asset/ETH feeds. To address this problem and enhance the stability of the Aave platform, we propose to change the price feeds for wstETH to a [custom price adapter](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CLSynchronicityPriceAdapterPegToBase.sol) that calculates the `wstETH / ETH / USD` price and is based on Chainlink's feeds under the hood. The same issue applies to the `WBTC` on Aave v2.
+The current price feeds for the LSTs can result in artificial volatility due to de-synchronization between different asset/PEG feeds. The same issue applies to the `WBTC` on Aave v2.
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:20]:
```suggestion
The current price feeds for certain ETH and MATIC LSTs can result in artificial volatility due to de-synchronization between different asset/PEG feeds.
Additionally, the Aave community already pre-approved pricing `WBTC` on Aave v2 based on a WBTC feed, and not assuming that its price is the one provided by BTC/ETH.
```

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 5]
```diff
@@ -1,20 +1,30 @@
...
+To address this problem and enhance the stability of the Aave platform, we propose to change the price feeds for wstETH to a [custom price adapter](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CLSynchronicityPriceAdapterPegToBase.sol) that calculates the `wstETH / ETH / USD` price and is based on Chainlink's feeds under the hood.
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:21]:
```suggestion
To address this problem and enhance the stability of the Aave platform, we propose to change the price feeds for wstETH on Optimism and Arbitrum to a [custom price adapter](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CLSynchronicityPriceAdapterPegToBase.sol) that calculates the `wstETH / ETH / USD` price and is based on Chainlink's feeds under the hood.
```

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 6]
```diff
@@ -1,20 +1,30 @@
...
+
+WBTC price feed will use the same [adapter implementatin](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CLSynchronicityPriceAdapterPegToBase.sol), but with the `WBTC / BTC / ETH` inside.
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:23]:
```suggestion
WBTC price feed will use the same [adapter implementation](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/CLSynchronicityPriceAdapterPegToBase.sol), but with the `WBTC / BTC / ETH` inside.
```

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 7]
```diff
@@ -1,20 +1,30 @@
...
+
+MaticX and stMatic price feeds will be swapped for a [special adapter](https://github.com/bgd-labs/cl-synchronicity-price-adapter/blob/main/src/contracts/MaticSynchronicityPriceAdapter.sol), which utilizes on-chain rate.
 
-This approach is already used for the wstETH on Polygon and WBTC on Aave V3.
+The same approach is already used for the wstETH on Polygon and WBTC on Aave V3.
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:27]: Probably better this reference on the appropriate paragraphs, not here

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 8]
```diff
@@ -26,18 +36,31 @@ Upon execution, the proposal will:
...
 - use Config Engine to update `wstETH` price feed to [0x05225Cd708bCa9253789C1374e4337a019e99D56](https://optimistic.etherscan.io/address/0x05225cd708bca9253789c1374e4337a019e99d56) on Optimism
 - use Config Engine to update `wstETH` price feed to [0x3105C276558Dd4cf7E7be71d73Be8D33bD18F211](https://arbiscan.io/address/0x3105c276558dd4cf7e7be71d73be8d33bd18f211) on Arbitrum
+- use Config Engine to update `MATICX` price feed to [0x0e1120524e14Bd7aD96Ea76A1b1dD699913e2a45](https://arbiscan.io/address/0x3105c276558dd4cf7e7be71d73be8d33bd18f211) on Polygon
+- use Config Engine to update `stMATIC` price feed to [0xEe96b77129cF54581B5a8FECCcC50A6A067034a1](https://arbiscan.io/address/0x3105c276558dd4cf7e7be71d73be8d33bd18f211) on Polygon
 
 ## Security and additional considerations
 
 Custom price adapters are already widely used in the system for price-correlated assets and were [reviewed by the auditors](https://github.com/bgd-labs/cl-synchronicity-price-adapter).
 
+- **MaticX Historical Feeds Comparison**: Comparing the answers from the current feed with the new one for the last 280 days (since CL feed was deployed) with a 4-hour step gave the following results:
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:46]: Here you can add that we have also double-checked all points of centralization of MaticX, to verify that no on-chain manipulation of the rate is possible, discounting permissioned actors

File: content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md [hunk 9]
```diff
@@ -26,18 +36,31 @@ Upon execution, the proposal will:
...
+
+  - 35.91% of results differ for less than 0.1%
+  - 62.06% have between 0.1% and 0.5% difference
+  - 2.03% is between 0.5% and 1%
+
+- **stMatic Historical Feeds Comparison**: Comparing the answers from the current feed with the new one for the last 327 days (since CL feed was deployed) with a 4-hour step gave the following results:
```
Comment (eboadom) [inline content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md:52]: Same as with MaticX about verification of on-chain manipulations

**Review by eboadom — APPROVED** [review]

**Pull request merged**